Source/Core/Core/Boot/Boot.h

```diff
@@ -23,10 +26,53 @@ struct RegionSetting
   const std::string code;
 };
 
+struct BootParameters
+{
+  struct Disc
+  {
+    std::string image_path;
```

Comment:
You're right, I have also removed the prefix for the other param structs (except for NAND, where it isn't obvious what the path is, since it should be a title ID normally).